examples/website/terrain-extension/README.md

```diff
@@ -27,3 +29,5 @@ Mapbox's [terrain API](https://docs.mapbox.com/help/troubleshooting/access-eleva
 
 To use other data sources, check out
 the [documentation of TerrainLayer](../../../docs/api-reference/geo-layers/terrain-layer.md).
+
+Data source for routes was prepared and processed using Tour de France GPX data freely available to download from https://www.cyclingstage.com/tour-de-france-2023-gpx/. 
```

Comment:
@cayetanobv I think to be safe we need to know what the license is

Reply:
There is no license at all on the website. They are a non-profit website that provides this information to the public. To be sure I'm going to write them an email asking for the use we want: "to support an example in an open source software project, properly citing them as the data source".